### PR TITLE
ENH: add tool to update Python repositories to the latest standards

### PR DESCRIPTION
**Pull request by klauer**
Automation tools for updating our Python/TwinCAT repositories to the latest standards

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+black
+jinja2
+pycln
+pyupgrade
+toml
```
Comment (tangkong) [inline requirements.txt:5]:
```suggestion
apischema
black
jinja2
pre-commit
pycln
pyupgrade
toml
```

Comment (klauer) [reply requirements.txt:5]: Thanks!

**Review by tangkong — APPROVED** [review]
Everything here looks good to me, including the fixes that came up in migrating lightpath over.  👍 

I imagine there'll be small tweaks in the future but this gets us so much of the way there 🙏

**Review by ZLLentz — APPROVED** [review]
This seems really comprehensive from what I've seen of the outputs

**Comment by klauer** [comment]
Oops, pre-commit failure caused me to erroneously make 2 commits into one in [d2cde41](https://github.com/pcdshub/pcds-python-migration-tools/pull/3/commits/d2cde415feb8616fb806647e3565199c4b90fc80) (should have been 'bump cookiecutter ref' then 'pre-commit autoupdate').

This repository isn't held to any real standards other than my own (as one might guess by above reviews and lack of PR text), so let's just go with it.

Merging this and moving back to the other PR.

**Pull request merged**